src/fusion.cpp

```diff
@@ -44,6 +44,20 @@
 #define MIOPEN_CHECK(x)          \
     if(x != miopenStatusSuccess) \
         return x;
+
+enum class ConvDirection
```

Comment:
Please check if we really need this for fusions.

Reply:
Since, I am reusing `ConvArgsForMIOpenDriver` that was defined in [src/convolution_api.cpp](https://github.com/ROCmSoftwarePlatform/MIOpen/blob/bg/LWPMIOPEN-149_re_worked/src/convolution_api.cpp), I do have to at least forward declare `enum class ConvDirection;`.

Reply:
Please see another review.

Reply:
[Resolved]